docs/manual/index.html

```diff
@@ -39,6 +39,7 @@
       <li><a href="#customize">Customizing Randoop's behavior to your application</a>
         <ul>
           <li><a href="#command-line-options">Command-line options</a></li>
+          <li><a href="#specifications">Specifying method and constructor behavior</a></li>
```

Comment:
Why does this pull request contain changes to docs/manual?
I would expect it to contain changes only to src/docs/manual, and docs/manual would be updated only when we make a release.